oidc-sso-sample/oidc-jks-loader/pom.xml

```diff
@@ -5,10 +5,11 @@
     <parent>
         <groupId>org.wso2.samples.is</groupId>
         <artifactId>oidc-sso-sample</artifactId>
-        <version>2.0.1-SNAPSHOT</version>
+        <version>3.0.0-m2-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+    <version>${parent.version}</version>
```

Comment:
Remove this one. The child pom gets parent version by default

Reply:
Will do :)